media-gfx/openvdb/openvdb-4.0.0.ebuild

```diff
@@ -0,0 +1,77 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $id$
+
+EAPI="6"
+PYTHON_COMPAT=( python{2_7,3_4,3_5} )
+
+inherit eutils flag-o-matic cmake-utils python-single-r1
+
+DESCRIPTION="Libs for the efficient manipulation of volumetric data"
+HOMEPAGE="http://www.openvdb.org"
+
+SRC_URI="https://github.com/dreamworksanimation/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MPL-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+abi3-compat doc python"
+
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+COMMON_DEPEND="
+	sys-libs/zlib
+	>=dev-libs/boost-1.62:=[${PYTHON_USEDEP}]
+	media-libs/openexr:=
+	media-libs/glfw:=
+	x11-libs/libXi
+	x11-libs/libXrandr
+	x11-libs/libXinerama
+	x11-libs/libXcursor
+	dev-libs/jemalloc
+	python? ( dev-python/numpy[${PYTHON_USEDEP}] )
```

Comment:
so python seems somehow optional, yet `REQUIRED_USE` unconditionally includes it?